test/tools/utils.ts

```diff
@@ -354,6 +354,18 @@ export class TestBuilder {
   }
 }
 
+export function bufferToStream(buffer) {
```

Comment:
Unrelated moving of this function from the message stream unit test to test utils.